examples/prostate/prostate_2D/configs/prostate_ditto/config/config_train.json

```diff
@@ -0,0 +1,9 @@
+{
+  "learning_rate": 1e-3,
+  "ditto_learning_rate": 1e-3,
+  "ditto_lambda": 1.0,
+  "fedproxloss_mu": 0.0,
+  "cache_dataset": 1.0,
+  "dataset_base_dir": "PWD/../data_preparation/dataset_2D",
```

Comment:
use {datafolder} alias instead of hardcoded path.

Reply:
changed